data/human/culture conversations.txt

```diff
@@ -120,6 +120,35 @@ mission "Paradise Cleaner Incident"
 
 
 
+mission "Class Trip to Port"
+	minor
+	source
+		attributes "south"
+		not attributes "station"
+	to offer
+		random < 1
+	on offer
+		conversation
+			`A group of children wave at you as you step off your ship, marveling at the spacecraft on display as their teachers try and fail to hold their attention. You could see the fading of their uniform colors from afar as their group walked past, yet none of them looked one bit displeased. You wonder how many of them may eventually own their own spaceship and fly across the stars as you make your way into the spaceport itself.`
+				decline
+
+
+
+mission "Sojourn Sunday Sales"
+	minor
+	source
+		government "Free Worlds"
+		attributes "urban"
+		not attributes "station"
+	to offer
+		random < 1
+	on offer
+		conversation
+			`You blink in surprise as you enter the spaceport, watching throngs of people pushing their way through the human chains of queues wrapped around and around the commercial area. You look around and see advertisements upon advertisements of vacation locations, all at an absurdly high discount. Yet you couldn't help but notice that to the younger you, it'd still be a price you'd never have be willing to spend on such things.`
```

Comment:
Third "around" in a row feels a bit clumsy. Also, there's two constructs à la "ads upon ads", "around and around" close together, which feels a bit on the nose.

```suggestion
			`You blink in surprise as you enter the spaceport, watching throngs of people pushing their way through the human chains of queues wrapped around the commercial area. You scan your surroundings and notice advertisements upon advertisements of vacation locations, all at an absurdly high discount. Yet you cannot help but notice that to the younger you, it'd still be a price you'd never be willing to spend on such things.`
```

Reply:
It's meant to be slightly on the nose, I like this version though.